### PR TITLE
Prepare script

### PR DESCRIPTION
**Pull request by dnewh**
This adds a prepare script to build the project for local installs and publishing.

### CODE REVIEW

**Review by dplumb94 — DISMISSED** [review]

**Review by ltseeley — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "yarn rollup:build",
+    "prepare": "npm run rollup:build",
```
Comment (peterschwarz) [inline package.json]: Why does this not just call build?

**Review by peterschwarz — CHANGES_REQUESTED** [review]
Add more context to the commit messages.

File: package.json (hunk 1 above)
Comment (dnewh) [reply package.json]: that was a leftover alias. I added a commit to simplify this.

**Comment by dnewh** [comment]
@peterschwarz updated

**Review by peterschwarz — APPROVED** [review]

**Review by vaporos — APPROVED** [review]

**Pull request merged**